src/audio/igo_nr/igo_nr.c

```diff
@@ -69,179 +69,316 @@ static void igo_nr_lib_process(struct comp_data *cd)
 	}
 }
 
+static void *igo_cir_wrap(void *ptr, void *cir_buf_end, size_t cir_buf_size)
```

Comment:
this comment is for title, what's the sample rate for igo_nr? due to I saw the frame size set to 768, if it is 16ms, means only support 48k input?

Reply:
The topologies in topology1 are for 48 kHz. The only clue in code is the `#define IGO_FRAME_SIZE (768)` that would be equivalent to 16 ms.